MobileDashboard/force-app/main/default/classes/DashboardSettingsService.cls

```diff
@@ -184,9 +184,15 @@ public with sharing class DashboardSettingsService {
 
     private static boolean isAuthorized() {
         try {
-            return [SELECT Id FROM Profile WHERE Name = 'System Administrator'].Id == System.UserInfo.getProfileId();
+            List<Profile> adminProfiles = [SELECT Id FROM Profile WHERE PermissionsContentAdministrator = true];
+            for (Profile p : adminProfiles) {
+                if (p.Id == System.UserInfo.getProfileId()) {
```

Comment:
shouldn't this be `===` ?

Reply:
`Id` in Apex is considered primitive, so it's sufficient to use `==`.
Moreover, I get an error if I try to use `===`:
`Exact equality operator only allowed for reference types: Id`.